Dockerfile.release

```diff
@@ -0,0 +1,8 @@
+FROM tkxkd0159/goreleaserx:1.19.3
```

Comment:
Can't we use official goreleaser image?

Reply:
There is no official Docker image entirely appropriate for our situation. If we can use two spaces for docker images, I can migrate this image from my repository to line docker workspace.